src/Dialog/Dialog.js

```diff
@@ -141,7 +141,7 @@ function getStyles(props, context) {
       fontSize: dialog.bodyFontSize,
       color: dialog.bodyColor,
       padding: gutter,
-      paddingTop: props.title ? 0 : gutter,
+      paddingTop: 0,
```

Comment:
That's no longer needed. I have fixed this issue.
But this change is breaking the `Alert` example:
![capture d ecran 2016-04-25 a 22 25 59](https://cloud.githubusercontent.com/assets/3165635/14797804/c4b8fe66-0b34-11e6-930b-698c0faabee7.png)